src/Common/src/CoreLib/System/SpanHelpers.Byte.cs

```diff
@@ -37,13 +37,13 @@ public static int IndexOf(ref byte searchSpace, int searchSpaceLength, ref byte
 
             byte valueHead = value;
             ref byte valueTail = ref Unsafe.Add(ref value, 1);
-            int valueTailLength = valueLength - 1;
+            nuint valueTailLength = valueLength - 1;
 
             int index = 0;
             for (; ; )
             {
-                Debug.Assert(0 <= index && index <= searchSpaceLength); // Ensures no deceptive underflows in the computation of "remainingSearchSpaceLength".
-                int remainingSearchSpaceLength = searchSpaceLength - index - valueTailLength;
+                Debug.Assert(0 <= index && searchSpaceLength >= index); // Ensures no deceptive underflows in the computation of "remainingSearchSpaceLength".
```

Comment:
Why did you change this?

Reply:
I added the `>=(NUInt left, int right)` operator, but not the other way around (i.e. `<=(int left, NUInt right)`) to the NUint wrapper for netfx.

Reply:
I think we should add as many operators as necessary to NUInt. so we do not need unnatural workarounds like this.

Reply:
OK. I will go ahead and add the necessary combinations.